default.xml

```diff
@@ -12,6 +12,7 @@
 
   <project path="external/libnfc-nci" name="android_external_libnfc-nci" remote="sony-patches"/>
   <project path="external/toybox" name="android_external_toybox" remote="sony-patches"/>
+  <project path="frameworks/av" name="platform/frameworks/av" remote="sony-patches"/>
```

Comment:
Technically, this is our (hybrs) patch, not Sony's. So it would need a new repo and branch like so: https://github.com/mer-hybris/android/blob/d92562e9905c0cc264080e690a5cd0e4893260e3/default.xml#L5
Too late now, will be for future :)